client/src/components/Dashboard/Listing/Listing.tsx

```diff
@@ -7,38 +7,43 @@ import Rating from '@mui/material/Rating';
 import Divider from '@mui/material/Divider';
 import LocationOnIcon from '@mui/icons-material/LocationOn';
 import useStyles from './useStyles';
+import { Profile } from '../../../interface/Profile';
 
-interface Props {
-  id?: string;
-  username: string;
-  description: string;
-  photo: string;
-  rating: number;
-  location: string;
-  price: number;
-}
-
-const Listing = ({ username, description, photo, rating, price, location }: Props): JSX.Element => {
+const Listing = ({ firstName, lastName, description, photo, price, address }: Profile): JSX.Element => {
   const classes = useStyles();
   return (
     <Card sx={{ width: 250, maxWidth: 350, minHeight: 250 }} className={classes.profile}>
       <CardContent className={classes.profileInfo1} sx={{ padding: '10px 30px' }}>
         <Avatar sx={{ width: 100, height: 100 }} alt="sitter photo" src={photo} />
-        <Typography variant="h5">{username}</Typography>
-        <Rating name="read-only" value={rating} readOnly />
-        <Typography variant="subtitle2">{description}</Typography>
+        <Typography variant="h5">
+          {firstName} {lastName}
+        </Typography>
+        <Rating name="read-only" value={4} readOnly />
+        {description ? (
+          <Typography variant="subtitle2">{description}</Typography>
+        ) : (
+          <Typography variant="h6">...</Typography>
+        )}
       </CardContent>
       <Divider />
       <CardContent className={classes.profileInfo2}>
-        <Box display="flex" alignItems="center">
-          <LocationOnIcon htmlColor="#f14140" />
-          <Typography variant="body2" color="text.secondary">
-            {location}
+        {address ? (
+          <Box display="flex" alignItems="center">
+            <LocationOnIcon htmlColor="#f14140" />
+            <Typography variant="body2" color="text.secondary">
+              {address}
+            </Typography>
+          </Box>
+        ) : (
+          <Typography variant="h6">...</Typography>
```

Comment:
Why don't we consider querying our database for sitter profiles who have a valid address/price (we wouldn't want to display the sitters who don't have a price/location as it doesn't fit the project use case)